apps/www/components/Marketing/index.js

```diff
@@ -20,30 +20,40 @@ import Logo from './Logo'
 import Community from './Community'
 import Pledge from './Pledge'
 import { useMarketingPageQuery } from './graphql/MarketingPageQuery.graphql'
+import ClimateLabTeaser from '../Climatelab/FrontTeaser/ClimateLabTeaser'
 
 const Marketing = ({
   data: { loading: meLoading, error: meError, meGuidance },
 }) => {
   const hasActiveMembership = meGuidance && !!meGuidance.activeMembership
+  const hasSpecialAccessGrants = meGuidance?.accessGrants?.filter(
+    (grant) => grant.campaign.type === 'REDUCED',
+  ).length
   const { inNativeApp, inNativeIOSApp } = useInNativeApp()
 
   const { data, loading, error } = useMarketingPageQuery()
 
   return (
     <>
-      {!meLoading && meGuidance && !hasActiveMembership && !inNativeIOSApp && (
-        <Box>
-          <MainContainer>
-            <UserGuidance />
-          </MainContainer>
-        </Box>
-      )}
+      {/* TODO: should we display another box for climate users or should we just not display it like it's done here */}
+      {!meLoading &&
+        meGuidance &&
+        !hasActiveMembership &&
+        !hasSpecialAccessGrants &&
+        !inNativeIOSApp && (
+          <Box>
+            <MainContainer>
+              <UserGuidance />
+            </MainContainer>
+          </Box>
+        )}
       {meError && (
         <ErrorMessage error={meError} style={{ textAlign: 'center' }} />
       )}
       <ColorContextProvider colorSchemeKey='dark'>
         <Top carouselData={data.carousel} />
       </ColorContextProvider>
+      <ClimateLabTeaser />
```

Comment:
TODO: remove from this PR and add to seperate PR to be merged on monday-morning.